skpro/datatypes/_convert.py

```diff
@@ -121,9 +128,14 @@ def convert(
     if obj is None:
         return None
 
+    # if to_type is a list, we do the following:
+    # if on the list, then don't do a conversion (convert to from_type)
+    # if not on the list, we find and convert to first mtype that has same scitype
+    to_type = _get_first_mtype_of_same_scitype(
+        from_mtype=from_type, to_mtypes=to_type, varname="to_type"
+    )
+
     # input type checks
-    if not isinstance(to_type, str):
```

Comment:
the input check gets removed entirely - should we not check for str or lits of str instead? Might be missing in both `sktime` and `skpro`.